tests/test-cases/parallel-matrix/integration.parallel-matrix.test.ts

```diff
@@ -22,6 +22,17 @@ test("parallel-matrix <test-job>", async () => {
         chalk`{blueBright test-job [bar,pro]} {greenBright >} NAME:'bar' TIER:'pro' 6/7`,
         chalk`{blueBright test-job [beb]    } {greenBright >} NAME:'beb' TIER:'' 7/7`,
 
+        chalk`{black.bgGreenBright  PASS } {blueBright pre-job           }`,
```

Comment:
This test should be unaffected by a the new `parallel: <integer>` feature
Why do we need to add extra expectations to this test in this PR?

Reply:
i can split this to another PR. The job was actually failing because of the typo above.

Reply:
https://github.com/firecow/gitlab-ci-local/pull/867 @repomaa I've made a PR fixing the failing test. Good catch, btw :smile: